src/promptflow/promptflow/_core/tools_manager.py

```diff
@@ -277,9 +280,16 @@ def _load_tool_from_module(
             if k not in init_inputs:
                 continue
             if v.value_type != InputValueType.LITERAL:
-                raise ValueError(
-                    f"Input {k!r} for tool '{tool_name}' only supports literal values for initialization,"
-                    + f" got {v.serialize()!r}"
+                raise InputTypeMismatch(
+                    message_format=(
+                        "The input for flow is incorrect. The tool '{tool_name}' requires a literal value for "
```

Comment:
Please hint that it is for initialization.

Reply:
Fixed.